tests/tck/features/subgraph/subgraph.IntVid.feature

```diff
@@ -4,7 +4,7 @@
 # attached with Common Clause Condition 1.0, found in the LICENSES directory.
 Feature: Integer Vid subgraph
 
-  Background: Prepare space
```

Comment:
Why change this?

Reply:
I lookup the docs of pytest-bdd, it's not necessary to add this description in background.